pontoon/base/static/js/fluent_interface.js

```diff
@@ -920,10 +920,20 @@ $(function () {
     if (isLabel) {
       selector = '.attributes [data-id="label"]';
     }
+
+    // Build artificial FTL Message from textarea contents to only take
+    // TextElements into account when generating access key candidates.
+    // See bug 1447103 for more detals.
     $('#ftl-area ' + selector + ' textarea').each(function() {
-      content += $(this).val()
-        .replace(/{.*?}/g, '') // Remove placeables: bug 1447103
-        .replace(/\s/g, '');   // Remove whitespace
+      var message = 'key = ' + $(this).val();
+      var ast = fluentParser.parseEntry(message);
+
+      ast.value.elements.forEach(function (element) {
+        if (element.type === 'TextElement') {
+          content += element.value
+            .replace(/\s/g, ''); // Remove whitespace
+        }
+      });
```

Comment:
That's a good start! :)

Would be cool to get `parsePattern` in `fluent-syntax` to avoid having to create a fake message.